### PR TITLE
COOPR-540: Implementation of the Tenant and Provisioner CLI commands

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: coopr-cli/src/main/java/co/cask/coopr/shell/command/DeleteTenantCommand.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+package co.cask.coopr.shell.command;
+
+import co.cask.common.cli.Arguments;
+import co.cask.common.cli.Command;
+
+import java.io.PrintStream;
+
+import co.cask.coopr.client.TenantClient;
+import com.google.inject.Inject;
```
Comment (albertshau) [inline coopr-cli/src/main/java/co/cask/coopr/shell/command/DeleteTenantCommand.java:22]: imports look weird, can you order them normally

**Comment by ghost** [comment]
Imports ordered normally

File: coopr-cli/src/main/java/co/cask/coopr/shell/command/GetProvisionerCommand.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+package co.cask.coopr.shell.command;
+
+import co.cask.common.cli.Arguments;
+import co.cask.common.cli.Command;
+import co.cask.coopr.client.ProvisionerClient;
+import co.cask.coopr.shell.util.CliUtil;
+
+import java.io.PrintStream;
+import com.google.inject.Inject;
```
Comment (albertshau) [inline coopr-cli/src/main/java/co/cask/coopr/shell/command/GetProvisionerCommand.java]: all other PRs have com.google right after co.cask, a newline, and then the java imports.  Please check that your IDE is using the same settings as others.

Comment (ghost) [inline coopr-cli/src/main/java/co/cask/coopr/shell/command/GetProvisionerCommand.java]: I have changed IDE settings and the order imports

**Comment by albertshau** [comment]
there are some test failures though I don't see how they could be related to the recent changes

**Comment by ghost** [comment]
This test failure occurred in REST Client module, though I haven't made any changes to this module

**Comment by ghost** [comment]
By the way (If I am not mistaken), yesterday's tests failures fixed here:https://github.com/caskdata/coopr/pull/726

**Comment by albertshau** [comment]
lgtm

**Pull request merged**